src/objects.js

```diff
@@ -1,41 +1,35 @@
 const createPerson = (name, age) => {
-  // your code here
+  return {name: name, age: age};
 };
 
-const getName = object => {
-  // your code here
-};
+const getName = object => object.name;
 
-const getProperty = (property, object) => {
-  // your code here
-};
+const getProperty = (property, object) => object[property];
```

Comment:
Here's when it's useful to use square dotation over dot notation - when you're not sure about which property you're extracting from the object and you want to leave it as dynamic as possible! Keep this one in your notes!